SPECS/boost/boost.spec

```diff
@@ -2,8 +2,8 @@
 
 Summary:        Boost
 Name:           boost
-Version:        1.76.0
-Release:        2%{?dist}
+Version:        1.82.0
```

Comment:
Since this is a big jump, I think we should build its dependents as well using buddy-build to ensure nothing will break down the line.

I did a quick grep to find following direct dependents.

- SPECS:
azure-storage-cpp
blobfuse
ceph
cereal
cpprest
hyperscan
msgpack
mstflint
swig
systemtap
toml11
websocketpp

- SPECS-EXTENDED:
asio
clucene
cpp-hocon
dyninst
exempi
leatherman
libabw
libcdr
libcmis
libepubgen
libetonyek
libfreehand
libftdi
libmspub
libopenraw
libpagemaker
libqxp
librevenge
libvisio
libwpd
libzmf
mdds
ptlib
sord

Reply:
Sure! I'll get one kicked off for AMD64 and ARM64.